skimage/restoration/_denoise.py

```diff
@@ -355,3 +361,93 @@ def denoise_tv_chambolle(im, weight=50, eps=2.e-4, n_iter_max=200,
         raise ValueError('only 2-d and 3-d images may be denoised with this '
                          'function')
     return out
+
+
+def _sigma_est_dwt(detail_coeffs, distribution='Gaussian'):
+    """
+    Calculation of the robust median estimator of the noise standard
+    deviation [1]_.
+
+    Parameters
+    ----------
+    detail_coeffs : ndarray
+        The detail coefficients corresponding to the discrete wavelet
+        transform of an image.
+    distribution : str
+        The underlying noise distribution.
+
+    Returns
+    -------
+    sigma : float
+        estimated noise standard deviation.  See [1]_
```

Comment:
You might want to tell people to look at section 4.2 of the paper. It's rather long. Also: period at the end of this sentence.